target/linux/ar71xx/image/legacy-devices.mk

```diff
@@ -427,6 +427,12 @@ define LegacyDevice/dLAN_pro_1200_ac
 endef
 LEGACY_DEVICES += dLAN_pro_1200_ac
 
+define LegacyDevice/DW33D
+  DEVICE_TITLE := DomyWifi DW33D
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-ledtrig-usbdev kmod-ath10k
+endef
+LEGACY_DEVICES += DW33D
```

Comment:
Using the old build code image size is 16 MByte.
Using the old build code to build the image without this error message.
It's a message from the update webpage of the legacy firmware.